panel/src/components/Forms/Fieldset.vue

```diff
@@ -16,13 +16,16 @@
 							:is="'k-' + field.type + '-field'"
 							v-if="hasFieldType(field.type)"
 							:ref="fieldName"
-							v-model="value[fieldName]"
+							:disabled="disabled || field.disabled"
 							:form-data="value"
 							:name="fieldName"
 							:novalidate="novalidate"
-							:disabled="disabled || field.disabled"
+							:value="value[fieldName]"
 							v-bind="field"
-							@input="$emit('input', value, field, fieldName)"
+							@input="
```

Comment:
this looks to me like we should put it into an onInput method instead.

Reply:
👍